### PR TITLE
{lib}[GCCcore/8.3.0] libepoxy v1.5.4

### PR DESCRIPTION
**Pull request by lexming**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
login2.cerberus.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6126 CPU @ 2.60GHz, Python 2.7.5
See https://gist.github.com/bfdcb08f42c3d415b76cfb8a3bcda03c for a full test report.

**Comment by bartoldeman** [comment]
Test report by @bartoldeman
**FAILED**
Build succeeded for 0 out of 2 (1 easyconfigs in this PR)
lg-1r17-n02 - Linux centos 6.6, Intel(R) Xeon(R) CPU E5-2650 0 @ 2.00GHz, Python 2.6.6
See https://gist.github.com/48872d6007fa78e0f97e00e769d781c1 for a full test report.

**Comment by bartoldeman** [comment]
Test report by @bartoldeman
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in this PR)
lg-1r17-n02 - Linux centos 6.6, Intel(R) Xeon(R) CPU E5-2650 0 @ 2.00GHz, Python 2.6.6
See https://gist.github.com/233f7278a52f973a4337b48664f585f4 for a full test report.

File: easybuild/easyconfigs/l/libepoxy/libepoxy-1.5.4-GCCcore-8.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+name = 'libepoxy'
+version = '1.5.4'
+
+homepage = 'https://github.com/anholt/libepoxy'
+description = "Epoxy is a library for handling OpenGL function pointer management for you"
+
+toolchain = {'name': 'GCCcore', 'version': '8.3.0'}
+
+github_account = 'anholt'
```
Comment (bartoldeman) [inline easybuild/easyconfigs/l/libepoxy/libepoxy-1.5.4-GCCcore-8.3.0.eb]: This looks suspicious, otherwise lgtm

Comment (bartoldeman) [reply easybuild/easyconfigs/l/libepoxy/libepoxy-1.5.4-GCCcore-8.3.0.eb]: I take it back, it's just what is put in the URL!

**Comment by bartoldeman** [comment]
Going in, thanks @lexming!

**Pull request merged**